ui/app/components/input-search.js

```diff
@@ -0,0 +1,18 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class inputSelect extends Component {
```

Comment:
Is this component used anywhere?

Reply:
It's the fallback component in the situation with the get-credentials-card where the user doesn't have permissions other than to type in a role name.  The naming of the class is wrong however, I'll go ahead and change that. `@fallbackComponent='input-search'` (see `get credentials-card.hbs`)